PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,16 @@
-[Explain what your PR does here]
+<!--
+  - Thanks for submitting code to Oppia! Please fill out the following as part of
+  - your pull request so we can review your code more easily.
+  -->
 
-**Checklist**
+## Explanation
+<!--
+  - Explain what your PR does and please include #bugnum in explanation.
+  -->
+
+## Checklist
 - [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes.
+- [ ] The PR explanation includes #bugnum.
```

Comment:
The PR explanation includes the words "Fixes #bugnum: ...".

Reply:
Done!